### PR TITLE
Bernick reponse changes

### PR DESCRIPTION
**Pull request by davidbernick**
Creating a new fresh PR.

### CODE REVIEW

File: AAI/AAIConnectProfile.md [hunk 1]
```diff
@@ -323,15 +321,13 @@ Payload:
...
 [“ga4gh”] : indicates that some RI claims are available beyond what is included in the access token but does not indicate which ones.
 [“ga4gh.ControlledAccessGrants”, “ga4gh.AffiliationAndRoles”] : indicates that only those two specific RI claims that exist within the “ga4gh” claim object would have additional content not included within the access token.
 
--   `<ga4gh-spec-claims>`: Claims included as part of a GA4GH standard specification based on the scopes provided. This content MAY be incomplete (i.e. a subset of data elements) and more may be fetched as indicated by ga4gh_userinfo_claims. A non-normative example of `<ga4gh-spec-claims>` is: "ga4gh": {[ga4gh claims](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE)}
```
Comment (cdvoisin) [inline AAI/AAIConnectProfile.md]: If you are leaving partial claims option on access tokens, then perhaps add something like this back in:

`<ga4gh-spec-claims>`: See description in /userinfo response below.

File: AAI/AAIConnectProfile.md [hunk 2]
```diff
@@ -346,6 +342,8 @@ Only the GA4GH claims truly must be as proscribed here. Refer to OIDC Spec for m
  \<ga4gh-spec-claims\>
 }
 ```
+-   `<ga4gh-spec-claims>`: Claims included as part of a GA4GH standard specification based on the scopes provided. This content MAY be incomplete (i.e. a subset of data elements) and more may be fetched as indicated by ga4gh_userinfo_claims. A non-normative example of `<ga4gh-spec-claims>` is: "ga4gh": {[ga4gh claims](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE)}
```
Comment (cdvoisin) [inline AAI/AAIConnectProfile.md]: Update RI claims spec links like this to be: https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/RI_Claims_V1.md

Comment (davidbernick) [reply AAI/AAIConnectProfile.md]: Can you do a PR to this branch? I'll merge.

File: AAI/AAIConnectProfile.md (hunk 1 above)
Comment (davidbernick) [reply AAI/AAIConnectProfile.md]: done

File: AAI/AAIConnectProfile.md (hunk 2 above)
Comment (davidbernick) [reply AAI/AAIConnectProfile.md]: done

**Pull request merged**